contracts/core/Registry.sol

```diff
@@ -71,6 +73,12 @@ contract Registry is AccessControl {
         return identitiesById[identityId];
     }
 
+    /// @notice Retrieve identity's pending owner
+    /// @param identityId The identityId of the identity
+    function getIdentityPendingOwner(bytes32 identityId) public view returns (address) {
+        return identityIdToPendingOwner[identityId];
+    }
+
```

Comment:
I don't think you need this getter honestly. Already have the mapping public, and it's not something that'll really be accessed.